cmd/crank/xpkg/xpkg.go

```diff
@@ -20,21 +20,25 @@ package xpkg
 // Cmd contains commands for interacting with xpkgs.
 // TODO(lsviben) add the rest of the commands from up (batch, xpextract).
 type Cmd struct {
-	Login   loginCmd   `cmd:"" help:"Login to the default package registry (xpkg.upbound.io)."`
-	Logout  logoutCmd  `cmd:"" help:"Logout of the default package registry (xpkg.upbound.io)."`
-	Build   buildCmd   `cmd:"" help:"Build a package, by default from the current directory."`
-	Push    pushCmd    `cmd:"" help:"Push a package, by default to xpkg.upbound.io."`
-	Install InstallCmd `cmd:"" help:"Install a package."`
-	Update  UpdateCmd  `cmd:"" help:"Update an installed package."`
+	// Keep subcommands sorted alphabetically.
+	Build   buildCmd   `cmd:"" help:"Build a new package."`
+	Install installCmd `cmd:"" help:"Install a package from a package registry."`
+	Login   loginCmd   `cmd:"" help:"Login to the default package registry."`
+	Logout  logoutCmd  `cmd:"" help:"Logout of the default package registry."`
+	Push    pushCmd    `cmd:"" help:"Push a package to a package registry."`
```

Comment:
```suggestion
	Install installCmd `cmd:"" help:"Install a package previously pushed to a registry."`
	Login   loginCmd   `cmd:"" help:"Login to a registry."`
	Logout  logoutCmd  `cmd:"" help:"Logout of the current registry."`
	Push    pushCmd    `cmd:"" help:"Push a package to a registry."`
```

Reply:
I made some of these updates. It now looks like this:

```
Commands:
  xpkg build      Build a new package.
  xpkg install    Install a package in a control plane.
  xpkg login      Login to the default package registry.
  xpkg logout     Logout of the default package registry.
  xpkg push       Push a package to a registry.
  xpkg update     Update a package in a control plane.
```

My rationale:

* I think login and logout really only work for xpkg.upbound.io, with the exception of for example the dev Upbound registry. AFAIK the commands don't work to login to index.docker.io for example. I'm tempted to actually say "Login to the xpkg.upbound.io package registry" but stuck with what we have for now.
* I glossed over where packages are installed from at this level and talked about where they're installed to, which I think is more important to call out. Install and update are the only `xpkg` commands that actually operate on a control plane.

Reply:
Looks good, sounds a good abstraction level to talk about control planes. 👍